Bottom: 9fbd5a9eb19f822e1fd393f1d441c7b75aa9b605
Top:    6656b3dabea0f1415107cbeea7b33d74ed35036c
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-01-09 14:33:27 +0530

Refresh of sched-core-add-manual

---

diff --git a/kernel/sched/sched.h b/kernel/sched/sched.h
index 84b46a992e11..841d78b6702f 100644
--- a/kernel/sched/sched.h
+++ b/kernel/sched/sched.h
@@ -2497,3 +2497,6 @@ static inline void membarrier_switch_mm(struct rq *rq,
 {
 }
 #endif
+
+#define lenient_latency(attr) \
+	((attr)->latency_tolerance == MAX_LATENCY_TOLERANCE)
